calculator.py

```diff
@@ -44,7 +44,7 @@ def calc(k):
 
     print ("\nScientific Calculator\nEg: pi * sin(90) - sqrt(81)")
 
-    k = input("\nWhat is ")
+    k = raw_input("\nWhat is ") # Using input() function is causing NameError. Changing it to raw_input() fixes this.
```

Comment:
In python 2, raw_input will always return a string, however input will treat it as an expression.
In Python 3, input returns a string. Maybe this helps!